draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -145,19 +159,24 @@ Selective disclosure:
 :  Process of a Holder disclosing to a Verifier a subset of claims contained in a claim set issued by an Issuer.
 
 Selectively Disclosable JWT (SD-JWT):
-:  A composite structure, consisting of an Issuer-signed JWT (JWS, [@!RFC7515]), Disclosures, and optionally a Key Binding JWT
- that supports selective disclosure as defined in this document. It can contain both regular claims and digests of selectively-disclosable claims.
+:  A composite structure, consisting of an Issuer-signed JWT (JWS, [@!RFC7515]) and zero or more Disclosures, which
+   supports selective disclosure as defined in this document. It can contain both regular claims and digests of selectively-disclosable claims.
 
 Disclosure:
 :  A combination of a salt, a cleartext claim name (present when the claim is a name-value pair and absent when the claim is an array element), and a cleartext claim value, all of which are used to calculate a digest for the respective claim.
 
 Key Binding:
 :  Ability of the Holder to prove legitimate possession of an SD-JWT by proving
-  control over the same private key during the issuance and presentation. An SD-JWT with Key Binding contains
-  a public key, or a reference to a public key, that matches to the private key controlled by the Holder.
+  control over the same private key during the issuance and presentation. To be compatible with Key Binding, an SD-JWT MUST contain
+  the public key corresponding to the private key controlled by the Holder (or a reference to this public key).
+
+Key Binding JWT (KB-JWT):
+:  A JWT for proving Key Binding as defined in (#kb-jwt).  A Key Binding JWT is
+   said to "cover" a particular SD-JWT if its payload includes a hash of the
+   in its `sd_hash` claim.
 
-Key Binding JWT:
-:  A JWT for proving Key Binding as defined in (#kb-jwt).
+Selectively Disclosable JWT with Key Binding (SD-JWT-KB):
+: A composite structure, comprising an SD-JWT and a Key Binding JWT covering the SD-JWT.
```

Comment:
```suggestion
: A composite structure, comprising an SD-JWT and a Key Binding JWT.
```
not sure what `covering the SD-JWT` is supposed to mean..

Reply:
It's not my most favorite term but the "Key Binding JWT" definition right above does indeed say 'A Key Binding JWT is said to "cover" a particular SD-JWT if its payload includes a hash of the in its `sd_hash` claim.'

Reply:
it's not intuitive, at least to me... Can we simply say `tied to that SD-JWT` (I modified the suggestion above)

Reply:
Editors discussed this on a call and agree that we prefer saying "tied to". I'll make the corresponding comments/suggestions.

Reply:
Those corresponding comments/suggestions aren't quite as straightforward as I'd expected. But it's done.

Reply:
I don't love "tied to", but I can live with it.